projects/admin/src/app/menu/service/library-switch-menu.service.ts

```diff
@@ -77,39 +77,45 @@ export class LibrarySwitchMenuService {
         .totalHits(results.hits.total) > 0 ? results.hits.hits : []
       ),
       map((result: any) => {
-        const libraries = {};
+        const libraries: ILibraryMenu[] = [];
         result.map((library: any) => {
-          libraries[library.metadata.pid] = library.metadata;
+          libraries.push({
+            id: library.metadata.pid,
+            name: library.metadata.name,
+            metadata: library.metadata
+          });
+        });
+        // Sort libraries by name asc.
+        return libraries.sort((first: ILibraryMenu, second: ILibraryMenu) => {
+          return first.name >= second.name ? 1 : -1
```

Comment:
you should use `localeCompare` function to compare string.
`return fist.name.localeCompare(second.name)`
Also, it's convenient to use "a" & "b" as variable name for comparaison methods.